pattern-1/marathon-applications/wso2apim.json

```diff
@@ -0,0 +1,107 @@
+{
+  "id": "wso2/apim",
+  "cmd": null,
+  "cpus": 1,
+  "mem": 2048,
+  "disk": 0,
+  "instances": 1,
+  "acceptedResourceRoles": [
+    "*"
+  ],
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "forcePullImage": false,
+      "image": "wso2am-dcos:2.1.0",
+      "parameters": [],
+      "privileged": false
+    },
+    "volumes": [
+      {
+        "containerPath": "/home/wso2carbon/wso2am-2.1.0/repository/conf/carbon.xml",
+        "hostPath": "/volumes/wso2/apim-analytics/repository/conf/carbon.xml",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/home/wso2carbon/wso2am-2.1.0/repository/conf/api-manager.xml",
+        "hostPath": "/volumes/wso2/apim-analytics/repository/conf/api-manager.xml",
```

Comment:
The `hostPath` should be `/volumes/wso2/apim/...`

Reply:
Yes, that's correct @msmshariq! I have fixed this with the latest commits. Thanks!